src/wallet/rpcdump.cpp

```diff
@@ -76,15 +76,16 @@ UniValue importprivkey(const UniValue& params, bool fHelp)
 {
     if (!EnsureWalletIsAvailable(fHelp))
         return NullUniValue;
-    
-    if (fHelp || params.size() < 1 || params.size() > 3)
+
+    if (fHelp || params.size() < 1 || params.size() > 4)
         throw runtime_error(
-            "importprivkey \"bitcoinprivkey\" ( \"label\" rescan )\n"
+            "importprivkey \"bitcoinprivkey\" ( \"label\" rescan timestamp)\n"
             "\nAdds a private key (as returned by dumpprivkey) to your wallet.\n"
             "\nArguments:\n"
             "1. \"bitcoinprivkey\"   (string, required) The private key (see dumpprivkey)\n"
             "2. \"label\"            (string, optional, default=\"\") An optional label\n"
             "3. rescan               (boolean, optional, default=true) Rescan the wallet for transactions\n"
+            "4. timestamp            (numeric, optional, default=0) Rescan starts at timestamp\n"
```

Comment:
@ruimarinho: note that `1231006505` is only the timestamp of the [mainnet](https://github.com/bitcoin/bitcoin/blob/e3f13ddc54142788261e3e0a16386cfcefb9bdc5/src/chainparams.cpp#L92) genesis block, whereby it's `1296688602` for [testnet](https://github.com/bitcoin/bitcoin/blob/e3f13ddc54142788261e3e0a16386cfcefb9bdc5/src/chainparams.cpp#L165) and [regtest](https://github.com/bitcoin/bitcoin/blob/e3f13ddc54142788261e3e0a16386cfcefb9bdc5/src/chainparams.cpp#L226).

I'm not sure, how the time might be further affected by [setmocktime](https://github.com/bitcoin/bitcoin/blob/e0862c0933bc809ef1087e3e0a3ee8cd76b04ac9/src/rpcmisc.cpp#L369), so I mention it just for completeness.